enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -156,12 +160,31 @@ def GetUniqueVerificationKey(self, **params):
         workload_id = "kme-uid"
         verification_key_nonce = params["nonce"]
         in_data = json.dumps({"nonce": verification_key_nonce})
-        wo_req = construct_wo_req(in_data, workload_id, self.encryption_key)
-
-        # @TODO : Trusted implementation to be integrated
-        # wo_response = self._execute_work_order(json.dumps(wo_req))
 
-        return ""
+        # Create session key and iv to sign work order request
+        session_key = crypto_utils.generate_key()
+        session_iv = crypto_utils.generate_iv()
+
+        wo_req = construct_wo_req(in_data, workload_id, self.encryption_key,
```

Comment:
No sure what are we trying to do here. My understanding is GetUniqueVerificationKey work order request should be created by WPE sent to KME listener. This work order request creation logic should be in WPE